src/components/HomePage/SearchArea/SearchArea.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { connect } from 'react-redux';
+import { addSnackbar } from '../../../actions/notificationActions';
+
+import JOB_TYPES from './jobTypes';
+
+import FormControl from '@material-ui/core/FormControl';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import Slider from '@material-ui/core/Slider';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import Collapse from '@material-ui/core/Collapse';
+
+import { makeStyles } from '@material-ui/styles';
+import searchAreaStyle from "./SearchArea.module.css";
+
+import SearchBar from "./SearchBar";
+import ShowAdvancedOptionsButton from './ShowAdvancedOptionsButton';
+import SliderValueTooltip from './SliderValueTooltip';
+
+const INITIAL_JOB_TYPE = "";
+const INITIAL_JOB_DURATION = 1;
+
+const useStyles = makeStyles({
+    wrapperInner: {
+        display: "grid",
+        "grid-template-columns": "1fr 1fr",
+        "grid-template-rows": "1fr",
+        "align-items": "center",
+        "grid-gap": "1em",
+    }
+});
+
+function SearchArea(props) {
+
+    const {addSnackbar, onSubmit} = props;
+    const classes = useStyles();
+    
+    // Set initial form values
+    const [searchValue, setSearchValue] = React.useState("");
+    const [advancedSearch, setAdvancedSearch] = React.useState(false);
+    const [jobType, setJobType] = React.useState(INITIAL_JOB_TYPE);
+    const [jobDuration, setJobDuration] = React.useState(INITIAL_JOB_DURATION);
+    
+    const toggleAdvancedOptions = () => {
+        if(advancedSearch) {
+            resetAdvancedFields();
+        }
+        setAdvancedSearch(!advancedSearch);
+    };
+
+    const resetAdvancedFields = () => {
+        setJobType(INITIAL_JOB_TYPE);
+        setJobDuration(INITIAL_JOB_DURATION);
+    };
+
+    const submitForm = e => {
+        e.preventDefault();
+
+        addSnackbar({
+            //mind the jobType.value || '' when passing value to api,
+            //because for simple search, the initial jobType value will be an empty string, which has no atrribute .value
+            message: `Search for: ${searchValue} :: Job type: ${jobType || ''} :: Job Duration: ${jobDuration}`,
+            options: {
+                variant: 'info',
+                anchorOrigin: {
+                    vertical: 'top',
+                    horizontal: 'right',
+                },
+            }
+        });
+
+        onSubmit();
+    };
+
+    const updateSearchValue = newVal => {
+        setSearchValue(newVal);
+    };
+    
+    const updateJobDuration = (_, val) => {
```

Comment:
Can `setJobDuration` be passed using some `.bind` shenanigans? Would reduce code clutter

Reply:
In the beginning I didn't see why, but that can't be done because the `Slider` `onChange` callback receives 2 arguments from the component. If it was the other way around, (we had a function with 2 args and the component only filled the second), we could bind the passed function with the first arg pre-filled

Reply:
Makes perfect sense! :shipit: